src/app/addition-template/addition-template.component.html

```diff
@@ -0,0 +1,3 @@
+<div *ngFor="let key of keys">
```

Comment:
doesn't it focus out on inputs here when you type something without `trackBy` on `*ngFor`